e2e/e2e_scripts_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// +build e2e_scripts
+
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
```

Comment:
The level of testing we're doing here should avoid importing any packages that are a part of `singularity` (especially internal ones). We really only want to test the executable and any other build artifacts that are installed. We want to avoid knowing anything about the internals of how `singularity` works, ie it should be considered a black box.

Examples: `buildcfg`, `sylog`, `user-agent`